cases/resources/casedb_instance_structure.xml

```diff
@@ -0,0 +1,28 @@
+<instance>
```

Comment:
I would call it "wrapper", honestly, just because I think that's the clearest way to illustrate that it's not there for a functional purpose